pkg/server/io/container_io.go

```diff
@@ -104,6 +115,10 @@ func (c *ContainerIO) Config() cio.Config {
 // Pipe creates container fifos and pipe container output
 // to output stream.
 func (c *ContainerIO) Pipe() {
+	if c.stdout == nil {
```

Comment:
could we add a comment here to explain the scenario where c.stdout could be nil?

Reply:
sure thing